admin_ui/src/components/EditRowForm.vue

```diff
@@ -133,17 +138,6 @@ export default Vue.extend({
             })
         },
     },
-    watch: {
-        "$route.params.tableName": async function () {
-            await Promise.all([
-                this.fetchData(),
-                this.$store.dispatch("fetchSchema", this.tableName),
-            ])
-        },
-        "$route.params.rowID": async function () {
-            await this.fetchData()
-        },
-    },
```

Comment:
Did this not work anymore? I think the original intention here was if someone hand modified the URL then the page would update appropriately.

Reply:
No. I had to remove all the route watchers because things are not working properly with them.

Reply:
What a pain. It might be possible with the composition API, but I can imagine a lot of people's code being broken by removing `$route`.

Reply:
You are actually right. I returned this piece of code in the last commit because the problem was actually in the FK `rowID` and I deleted this code before solving that issue because I didn't know what the problem was. Thanks for your suggestion.